docs/install/check-install-version.md

```diff
@@ -5,97 +5,33 @@ weight: 20
+version from the container images of your Knative controllers:
 
-This will return the description for the `knative-serving` controller; this
-information contains the link to the container that was used to install Knative:
-
-```yaml
----
-Pod Template:
-  Labels: app=controller
-  Annotations: sidecar.istio.io/inject=false
-  Service Account: controller
-  Containers:
-    controller:
-      # Link to container used to run the Knative Serving controller
-      Image: gcr.io/knative-releases/github.com/knative/serving/cmd/controller@sha256:59abc8765d4396a3fc7cac27a932a9cc151ee66343fa5338fb7146b607c6e306
-```
-
-Copy the full `gcr.io` link to the container and paste it into your browser. If
-you are already signed in to a Google account, you'll be taken to the Google
-Container Registry page for that container in the Google Cloud Platform console.
-If you aren't already signed in, you'll need to sign in to a Google account
-before you can view the container details.
-
-On the container details page, you'll see a section titled "Container
-classification," and in that section is a list of tags. The versions of Knative
-you have installed will appear in the list as `v0.1.1`, or whatever version you
-have installed:
-
-![Shows list of tags on container details page; v0.1.1 is the Knative version and is the first tag.](../images/knative-version.png)
-
-## Knative Eventing
-
-To check what version of Knative serving you have installed, enter the following
-command:
-
-```bash
-kubectl describe deploy eventing-controller --namespace knative-eventing
-```
-
-This will return the description for the `knative-eventing` controller; this
-information contains the link to the container that was used to install Knative:
-
-```yaml
----
-Pod Template:
-  Labels: app=eventing-controller eventing.knative.dev/release=devel
-  Service Account: eventing-controller
-  Containers:
-    eventing-controller:
-      # Link to container used to run the Knative Eventing controller
-      Image: gcr.io/knative-releases/github.com/knative/eventing/cmd/controller@sha256:85c010633944c06f4c16253108c2338dba271971b2b5f2d877b8247fa19ff5cb
-```
-
-Copy the full `gcr.io` link to the container and paste it into your browser. If
-you are already signed in to a Google account, you'll be taken to the Google
-Container Registry page for that container in the Google Cloud Platform console.
-If you aren't already signed in, you'll need to sign in to a Google account
-before you can view the container details.
-
-On the container details page, you'll see a section titled "Container
-classification," and in that section is a list of tags. The versions of Knative
-you have installed will appear in the list as `v0.1.1`, or whatever version you
-have installed:
-
-![Shows list of tags on container details page; v0.1.1 is the Knative version and is the first tag.](../images/knative-version.png)
+1. Query your Knative Controller to receive a URL.
```

Comment:
```suggestion
1. Query your Knative Controller to receive the `Image` URL. For example, you can run `kubectl describe deploy controller --namespace knative-serving` or `kubectl describe deploy eventing-controller --namespace knative-eventing`
```

Reply:
I did a quick edit. 
Also, Im okay with leaving this out too if you feel strongly about expecting nobody to need those details. 
Otherwise this looks great! Thanks @dgerd (LMK and ill merge)

Reply:
I like the edit. Lets leave them in few a little while longer, but we should consider removing this section after a few more releases.